docs/task_on_kart.rst

```diff
@@ -250,3 +250,25 @@ TaskOnKart.should_dump_supplementary_log_files
 Whether to dump supplementary files (task_log, random_seed, task_params, processing_time, module_versions) or not. Default is True.
 
 Note that when set to False, task_info functions (e.g. gokart.tree.task_info.make_task_info_as_tree_str()) cannot be used.
+
+
+Dump csv with encoding
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can dump csv file by impementing `Task.output()` method as follows: 
+
+.. code:: python
+
+    def output(self):
+        return self.make_target('file_name.csv')
+
+By default, csv file is dumped with `utf-8` encoding.
+
+If you want to dump csv file with other encoding, you can use `encoding` parameter as follows:
```

Comment:
```suggestion
If you want to dump csv file with another encoding, you can use `encoding` parameter as follows:
```

Reply:
I've changed to `other encodings`. Thank you!